### PR TITLE
Update zh-CN translations

### PR DESCRIPTION
**Pull request by bastengao**
Complement translations  for `zh-CN`.

### CODE REVIEW

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -92,10 +92,12 @@ zh-CN:
         expires_at: "过期日期"
         name: "名称"
         path: "路径"
+        promotion_category: "促销分类"
```
Comment (futhr) [inline config/locales/zh-CN.yml]: This key do not exists in Spree core `master` locale as a sub key (it do exist as a root key and as an ActiveRecord model). See: https://github.com/spree/spree/blob/master/core/config/locales/en.yml

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -377,28 +387,29 @@ zh-CN:
...
         configuration: "配置"
-        option_types:
+        option_types: "选项类型"
         orders: "订单"
         overview: "概览"
         products: "产品"
         promotions: "促销"
-        properties:
-        prototypes:
+        promotion_categories: "促销分类"
```
Comment (futhr) [inline config/locales/zh-CN.yml]: This key do not exist either.

Comment (bastengao) [inline config/locales/zh-CN.yml]: It is used in sidebar menu of spree_backend, see https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/shared/sub_menu/_promotion.html.erb#L3 . It is used as sub key in `admin/tab`, see https://github.com/spree/spree/blob/master/backend%2Fapp%2Fhelpers%2Fspree%2Fadmin%2Fnavigation_helper.rb#L30 .

File: config/locales/zh-CN.yml (hunk 1 above)
Comment (bastengao) [inline config/locales/zh-CN.yml]: `promotion_category ` do not exists in core en.yml file, but it is used in promotions form of spree_backend. See: https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/promotions/_form.html.erb#L35 . This label displays right with default translation in English, but wrong in Chinese.

File: config/locales/zh-CN.yml (hunk 2 above)
Comment (futhr) [inline config/locales/zh-CN.yml]: `plural_resource_name(Spree::PromotionCategory)`

File: config/locales/zh-CN.yml (hunk 1 above)
Comment (futhr) [inline config/locales/zh-CN.yml]: `Spree::PromotionCategory.model_name.human`

File: config/locales/zh-CN.yml (hunk 2 above)
Comment (futhr) [inline config/locales/zh-CN.yml]: Think you correct here actually, the tab key seems to be absent all over except from `:ru` and `:pt_BR`

File: config/locales/zh-CN.yml (hunk 1 above)
Comment (bastengao) [inline config/locales/zh-CN.yml]: That's it. What can I do in this pr?

**Comment by futhr** [comment]
Hi @bastengao, these commits has been cherry-picked into `master` branch. I removed the model relation translation `:promotion_category`. If this still remain an issue, then its a global issue that need to be resolved, but I was unable to find anything related to this.

**Pull request closed without merging**